protected/controllers/DayOneController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+class DayOneController extends Controller
+{
+
+    public function actionAbout()
+    {
+        $this->render('about');
+    }
+
+    public function actionError()
+    {
+        if ($error = Yii::app()->errorHandler->error) {
+            if (Yii::app()->request->isAjaxRequest) {
+                echo $error['message'];
+            } else {
+                $this->render('error', $error);
+            }
+        }
+    }
+
+    public function actionNumber()
+    {
+        $count_print = array();
+
+        for ($count = 1; $count <= 10; $count++) {
+            if ($count <= 9) {
+                $count_print[$count - 1] = $count." - ";
+            } else {
+                $count_print[$count - 1] = $count;
+            }
+        }
+
+        $this->render('number', array('count_print' => $count_print));

```

Comment:
Instead of declaring many arrays like `['count_print' => $count_print]`, you may want to check out PHP's `compact()` function :penguin: